src/agent/communicator/src/http_client.cpp

```diff
@@ -101,8 +101,7 @@ namespace http_client
                         reqParams.Endpoint,
```

Comment:
Maybe the variable `code` from https://github.com/wazuh/wazuh-agent/pull/355/files#diff-301a5da8240724f147370f285f68ff252a29505ea09942939dbec96f24034596R95 can be reused as you did in `PerformHttpRequest`.

Reply:
What is the difference between `boost::system::error_code` and `boost::beast::error_code`?